install-storageos/020-install-storageos.md

```diff
@@ -0,0 +1,100 @@
+
+First we need to install the StorageOS Operator using the following yaml manifest.
+
+`kubectl create -f https://github.com/storageos/cluster-operator/releases/download/v2.1.0/storageos-operator.yaml`{{execute}}
+
+Verify the Cluster Operator pod status
+
+`kubectl -n storageos-operator get pod -w`{{execute}}
+
+> The READY 1/1 indicates that `storageoscluster` resources can be created. Press `Ctrl+C` to continue once the pod is up.
+
+Create a Secret defining the StorageOS credentials. The username and password are just `storageos` encoded in base64.
+
+```
+kubectl create -f- <<END
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "storageos-api"
+  namespace: "storageos-operator"
+  labels:
+    app: "storageos"
+type: "kubernetes.io/storageos"
+data:
+  # echo -n '<secret>' | base64
+  apiUsername: c3RvcmFnZW9z
+  apiPassword: c3RvcmFnZW9z
+  # CSI Credentials
+  csiProvisionUsername: c3RvcmFnZW9z
+  csiProvisionPassword: c3RvcmFnZW9z
+  csiControllerPublishUsername: c3RvcmFnZW9z
+  csiControllerPublishPassword: c3RvcmFnZW9z
+  csiNodePublishUsername: c3RvcmFnZW9z
+  csiNodePublishPassword: c3RvcmFnZW9z
+END
+```{{execute}}
+
+This Secret is referenced from the StorageOS Cluster Resource below and will be used
+to create an admin user in StorageOS at bootstrap. Also to set up the CSI
+credentials that will be exchanged with the Kubernetes ControlPlane, so
+StorageOS and Kubernetes can communicate.
+
+StorageOS Installation.
+
+The following resource defines the StorageOS Cluster. Many options are available
+for configuration. For instance, note the `.spec.kvBackend.address` that indicates
+the etcd url, or the `.spec.namespace` that indicates where StorageOS will be
+installed. Other options could be configured such as `nodeSelectors` to
+indicate that the StorageOS Daemonset should run on a subset of nodes.
+
+```
+kubectl create -f- <<END
+apiVersion: "storageos.com/v1"
+kind: StorageOSCluster
+metadata:
+  name: "my-storageos-cluster"
+  namespace: "storageos-operator"
+spec:
+  secretRefName: "storageos-api" # Reference from the Secret created in the previous step
+  secretRefNamespace: "storageos-operator"  # Namespace of the Secret
+  namespace: "kube-system"
+  k8sDistro: "upstream"
+  images:
+    nodeContainer: "storageos/node:v2.0.0" # StorageOS version
```

Comment:
any reason we are keeping v2.0.0 here? The operator we are installing is v2.1.0

Reply:
Oh that's a typo, fixing it now.

Reply:
You will also need to add 
enableControllerExpandCreds: true  in the cluster definition
and the in the secret the username and password
csiControllerExpandUsername: c3RvcmFnZW9z
csiControllerExpandPassword: c3RvcmFnZW9z